packages/strapi/lib/core/plugins.js

```diff
@@ -88,6 +88,10 @@ module.exports = async function() {
     return;
   }
 
+  if (strapi.config.environment !== 'development') {
```

Comment:
Maybe we can add a check and a warning in the server if the file is missing. So the user would know that the front-end cannot work without the `plugins.json` file?

Reply:
Something like this https://github.com/strapi/strapi/pull/2518/commits/7d039644fae62e63994546783fade20158db72ac?